templates/systemd.override.j2

```diff
@@ -0,0 +1,10 @@
+[Service]
```

Comment:
Why is a custom systemd config required?  Do the official (pgdg) postgres /rh/centos) rpms not provide+install this?